### PR TITLE
Internal site search 010421a

### PR DESCRIPTION
**Pull request by jbranson000**
#1017

### CODE REVIEW

File: components/datatypes/internal-site-search.schema.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/internal-site-search",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Internal Site Search",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:tags": {
```
Comment (mjasrotia) [inline components/datatypes/internal-site-search.schema.json:13]: We talked about not having the industry tags in the data types right ?

File: components/datatypes/internal-site-search.schema.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          "title": "Autocomplete Value",
+          "type": "string",
+          "description": "For autocomplete scenarios, users sometimes abandon their search and click a specific term from the drop down. This value is used to track the specific terms clicked"
+        },
+        "xdm:autocompleteTypedValue": {
+          "title": "Autocomplete Typed Value",
+          "type": "string",
+          "description": "For autocomplete scenarios, users sometimes abandon their search and click a specific term from the drop down. This value tracks what the user started to type in order to generate the specific set of suggested search terms"
+        },
+        "xdm:type": {
```
Comment (mjasrotia) [inline components/datatypes/internal-site-search.schema.json:121]: This is to generic for a name. Should we rename this to "searchType" or something like that ?

Comment (jbranson000) [reply components/datatypes/internal-site-search.schema.json:121]: I've updated this in the next build.

File: components/datatypes/internal-site-search.schema.json (hunk 1 above)
Comment (jbranson000) [reply components/datatypes/internal-site-search.schema.json:13]: Correct - I've updated the next commit to reflect that, and removed it from the automated process.

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**